listenbrainz/webserver/static/js/src/user/UserFeedback.tsx

```diff
@@ -11,12 +11,13 @@ import {
   faHeartBroken,
   faThumbtack,
 } from "@fortawesome/free-solid-svg-icons";
-import { isNaN, get, clone, has } from "lodash";
+import * as _ from "lodash";
+import { clone, has, isNaN } from "lodash";
```

Comment:
you can add `get` to the named import line underneath and remove the `_.` prefix on lines 428 and 438, and then remove `import * as _ from "lodash";`